.github/workflows/dependabot-approve-merge.yml

```diff
@@ -24,14 +24,14 @@ concurrency:
 
 jobs:
   auto-approve-merge:
-    if: github.actor == 'dependabot[bot]'
-    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]' || github.actor == 'renovate[bot]'
+    runs-on: ubuntu-latest-low
```

Comment:
Does not exist on the libraries organization. Either revert to `ubuntu-latest` or convince sysadmins to also add the tagged runners here ;)

Reply:
reverted, for the sake of low-effort